src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -55,13 +57,17 @@ protected function configure()
   <info>php %command.full_name% framework</info>
   <info>php %command.full_name% FrameworkBundle</info>
 
+The <info>--format</info> option specifies the format of the configuration,
+this is either <comment>yaml</comment> or <comment>json</comment>.
+
+  <info>php %command.full_name% FrameworkBundle --format=json</info>
```

Comment:
```suggestion
  <info>php %command.full_name% framework --format=json</info>
```

nitpick:
the command example bellow uses the bundle config name... looks more consistent to me that this example remains in the same manner